versioned_docs/version-v2.9.0/troubleshooting/troubleshooting.md

```
@@ -160,3 +160,53 @@ devicePlugin:
 ```
 
 :::
+## Pod Stuck in Pending {#pod-stuck-in-pending}
```

markdownlint-cli2 (0.23.2)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Above

(MD022, blanks-around-headings)